content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,30 +5,161 @@ weight = 20
+            Kubeflow Katib
+          </a>
+        </td>
+        <td>
+          Model Optimization and AutoML
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/katib">
+            <code>kubeflow/katib</code>
+          </a>
+        </td>
+      </tr>
+      <tr>
+        <td>
+         <a href="https://kserve.github.io/website/master/admin/serverless/serverless">
+            KServe
+          </a>
+        </td>
+        <td>
+          Model Serving
+        </td>
+        <td>
+          <a href="https://github.com/kserve/kserve">
+            <code>kserve/kserve</code>
+          </a>
+        </td>
+      </tr>
+      <tr>
+        <td>
+         <a href="/docs/components/model-registry/installation/#installing-model-registry">
+            Kubeflow Model Registry
+          </a>
+        </td>
+        <td>
+          Model Registry
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/model-registry">
+            <code>kubeflow/model-registry</code>
+          </a>
+        </td>
+      </tr>
+      <tr>
+        <td>
+         <a href="/docs/components/training/user-guides/mpi/#installation">
+            Kubeflow MPI Operator
+          </a>
+        </td>
+        <td>
+          All-Reduce Model Training
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/mpi-operator">
+            <code>kubeflow/mpi-operator</code>
+          </a>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="/docs/components/pipelines/v2/installation/quickstart/">
+            Kubeflow Pipelines
+          </a>
+        </td>
+        <td>
+          ML Workflows and Schedules
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/pipelines">
+            <code>kubeflow/pipelines</code>
+          </a>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="https://github.com/kubeflow/spark-operator/tree/master?tab=readme-ov-file#installation">
+            Kubeflow Spark Operator
+          </a>
+        </td>
+        <td>
+          Data Preparation
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/spark-operator">
+            <code>kubeflow/spark-operator</code>
+          </a>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="/docs/components/training/installation/#installing-training-operator">
+            Kubeflow Training Operator
+          </a>
+        </td>
+        <td>
+          Model Training and Fine-Tuning
+        </td>
+        <td>
+          <a href="https://github.com/kubeflow/training-operator">
+            <code>kubeflow/training-operator</code>
+          </a>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+### Kubeflow Platform
+
+You can use one of the following methods to install Kubeflow Platform to get full suite of Kubeflow
+components bundled together with additional integration and management tools.
```

Comment:
We can link people to the introduction `"What is Kubeflow Platform?"` section, to help them understand the differences:

```suggestion
You can use one of the following methods to install Kubeflow Platform.

When deployed as a platform, Kubeflow provides a comprehensive set of tools for the entire ML lifecycle.
For more information about the platform, please see this [introduction](/docs/started/introduction/#what-is-kubeflow-platform).
```

Reply:
I can modify this as follows:
```
You can use one of the following methods to install [Kubeflow Platform](/docs/started/introduction/#what-is-kubeflow-platform)
to get full suite of standalone Kubeflow components bundled together with additional tools.
```

We already linked the introduction page at the beginning of this page.

Reply:
I think an active tense is better.

```
You can use one of the following methods to install the [Kubeflow Platform](/docs/started/introduction/#what-is-kubeflow-platform).
When deployed as a platform, Kubeflow provides a comprehensive set of tools for the entire ML lifecycle. 
```

Which is more similar to what you originally proposed anyway, just with a link to the introduction page.

Reply:
We have to say that all Standalone Kubeflow Components are included in the Kubeflow Platform.

Reply:
@andreyvelich I think that is implied. Also, its explicitly said in the `/docs/started/introduction/#what-is-kubeflow-platform` link if the user clicks.

The key "selling point" of the platform is the `When deployed as a platform, Kubeflow provides a comprehensive set of tools for the entire ML lifecycle.`.

Talking about `standalone components` in this heading detracts from this idea of using Kubeflow as a platform, I get that your interest is in Kubeflow not being a platform, but many users want a platform, and that's why we have two sections on this page.

Reply:
Ok, can you rephrase this message to explain that all Standalone Kubeflow Components will be included into Kubeflow Platform ?